Compiler/src/implementation/InstructionManager.cpp

```diff
@@ -0,0 +1,84 @@
+#include "../header/InstructionManager.h"
+
+using namespace std;
+
+string returnString = "";
+
+string InstructionManager::AddInstructionRO(RegisterOnlyInstruction RO, int R1, int R2, int R3, string comment)
```

Comment:
we should probably make an enumeration for the Registers. that way the paramater `int R1` would become `RegisterEnum R1` and so forth. This would prevent someone from trying to do something like `ADD 1, 1, 100` since register 100 isn't a thing obviously. We can do this tomorrow